packages/camera/camera/ios/Classes/FLTCam.h

```diff
@@ -60,6 +60,7 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)setExposureModeWithResult:(FLTThreadSafeFlutterResult *)result mode:(NSString *)modeStr;
 - (void)setFocusModeWithResult:(FLTThreadSafeFlutterResult *)result mode:(NSString *)modeStr;
 - (void)applyFocusMode;
+- (void)receivedImageStreamData;
```

Comment:
This needs a declaration comment, per style guide.

(I know almost nothing else in this header current has one, but existing style violations aren't a reason to perpetuate new style violations.)

Reply:
Done.
But, my English isn't very good, so please point out any oddities.